Gemfile

```diff
@@ -6,9 +6,7 @@ gem 'jira-ruby', :group => :development
 group :test do
   gem 'rspec'
   gem 'beaker', '~>1.21.0'
-  if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '0.12.1'
-  end
+  gem "beaker-hostgenerator", *location_for(ENV['BEAKER_HOSTGENERATOR_VERSION'] || "~> 0.7")
```

Comment:
@nwolfe I just pulled the same version that I found from master branch on this repo. I'll leave it up to @camlow325 if it should be upgraded here. Other than the fact that the other puppetserver branches are using `0.7.0` I don't see any reason not to use `0.8.0` here.